config/locales/en/document_types.yml

```diff
@@ -80,33 +80,33 @@ en:
 
     fatality_notice:
       description: Initial fatality notices and subsequent obituaries of forces and MOD personnel
-      
```

Comment:
These space fixes could be moved to their own commit.

Reply:
I think it's reasonable to have this change here, although I appreciate we should try to limit the scope of a commit if possible. There's an effort/reward ratio to changing this, and I'm not sure it's worth it in this case. 

As someone who likes to read through the sequence of commits, I'd prefer to see a slightly larger diff than read through an extra commit that may or may not do more than just fix whitespacing (never trust a commit title 😉 ).